packages/jira-adapter/test/filters/jsm_types_fetch_filter.test.ts

```diff
@@ -79,5 +95,86 @@ describe('jsmTypesFetchFilter', () => {
         new ReferenceExpression(projectInstance.elemID, projectInstance),
       )
     })
+    it('should change the deployment annotations of icon field in object type icon', async () => {
+      const objectTypeIconType = new ObjectType({
+        elemID: new ElemID(JIRA, 'ObjectTypeIcon'),
+        fields: {
+          icon: {
+            refType: BuiltinTypes.STRING,
+            annotations: { [CORE_ANNOTATIONS.UPDATABLE]: true },
+          },
+        },
+      })
+      elements.push(objectTypeIconType)
+      await filter.onFetch(elements)
+      expect(objectTypeIconType.fields.icon.annotations[CORE_ANNOTATIONS.UPDATABLE]).toBe(false)
+    })
+    it('should change conditionId refType to unknown in SLA types', async () => {
+      const slaTypes = slaTypeNames.map(
+        typeName =>
+          new ObjectType({
+            elemID: new ElemID(JIRA, typeName),
+            fields: {
+              conditionId: {
+                refType: BuiltinTypes.STRING,
+              },
+              name: {
+                refType: BuiltinTypes.STRING,
+                annotations: { [CORE_ANNOTATIONS.CREATABLE]: true, [CORE_ANNOTATIONS.UPDATABLE]: true },
+              },
+            },
+          }),
+      )
```

Comment:
nit: lets move it to be function
```suggestion
const createSlaType = (typeName: string): ObjectType => 
  new ObjectType({
    elemID: new ElemID(JIRA, typeName),
    fields: {
      conditionId: {
        refType: BuiltinTypes.STRING,
      },
      name: {
        refType: BuiltinTypes.STRING,
        annotations: {
          [CORE_ANNOTATIONS.CREATABLE]: true,
          [CORE_ANNOTATIONS.UPDATABLE]: true,
        },
      },
    },
  });

const slaTypes = slaTypeNames.map(createSlaType);
```